api/src/app.ts

```diff
@@ -38,6 +40,16 @@ initialize({
     'application/json': bodyParser.json({ limit: '50mb' }),
     'application/x-www-form-urlencoded': bodyParser.urlencoded({ limit: '50mb', extended: true })
   },
+  securityHandlers: {
+    Bearer: function (req, scopes) {
+      // return true // bypass authentication
+      return authenticate(req, scopes);
+    }
+  },
+  securityFilter: async (req, res) => {
+    const updatedReq = await applyApiDocSecurityFilters(req);
+    res.status(200).json(updatedReq['apiDoc']);
+  },
```

Comment:
Hey, sorry for the back and forth, I probably didn't describe the difference very well:

We need the `securityHandlers` part, but we don't need the `securityFilter` part. 

The `securityHandlers` part is what runs our authentication code (verifying the token, and user roles, etc), so thats for sure needed.

The `securityFilter` part only sort of does security stuff, but not really (the name isn't great). What it does is manipulate the api-doc to add/remove content (which you might choose to do for security reasons), but it doesn't actually add any security to the endpoints themselves.

Reply:
That being said, The current setup wont break anything.  Though it wont add anything either (just because the api-doc for the codes doesnt have any x-enum tags).  So could leave it for now.